Gruntfile.js

```diff
@@ -108,7 +110,8 @@ module.exports = function (grunt) {
                     '<%= config.lib %>/angular-bind-html-compile/angular-bind-html-compile.js',
                     '<%= config.lib %>/angular-gettext/dist/angular-gettext.js',
 
-                    'dist/**/*.js',
+                    'src/**/*.js',
```

Comment:
I changed this to make debugging easier during testing (otherwise it is uglifyed)